svm/src/transaction_processor.rs

```diff
@@ -861,6 +862,7 @@ impl<FG: ForkGraph> TransactionBatchProcessor<FG> {
                 );
             }
         }
+
```

Comment:
I'd prefer not adding empty lines, but it's ok to leave it in.

Reply:
Removed